eth/stagedsync/stage_mining_exec.go

```diff
@@ -96,23 +96,23 @@ func StageMiningExecCfg(
 // SpawnMiningExecStage
 // TODO:
 // - resubmitAdjustCh - variable is not implemented
-func SpawnMiningExecStage(s *StageState, txc wrap.TxContainer, cfg MiningExecCfg, sendersCfg SendersCfg, execCfg ExecuteBlockCfg, ctx context.Context, logger log.Logger) error {
+func SpawnMiningExecStage(s *StageState, txc wrap.TxContainer, cfg MiningExecCfg, sendersCfg SendersCfg, execCfg ExecuteBlockCfg, ctx context.Context, logger log.Logger, u Unwinder) error {
 	cfg.vmConfig.NoReceipts = false
 	chainID, _ := uint256.FromBig(cfg.chainConfig.ChainID)
 	logPrefix := s.LogPrefix()
 	current := cfg.miningState.MiningBlock
 	txs := current.PreparedTxs
 	noempty := true
-	var domains *state2.SharedDomains
 	var (
 		stateReader state.StateReader
 	)
 	stateReader = state.NewReaderV3(txc.Doms)
 	ibs := state.New(stateReader)
 	// Clique consensus needs forced author in the evm context
-	if cfg.chainConfig.Consensus == chain.CliqueConsensus {
-		execCfg.author = &cfg.miningState.MiningConfig.Etherbase
-	}
+	//if cfg.chainConfig.Consensus == chain.CliqueConsensus {
+	//	execCfg.author = &cfg.miningState.MiningConfig.Etherbase
+	//}
+	execCfg.author = &cfg.miningState.MiningConfig.Etherbase
```

Comment:
Should we here revise all consensuses? My 5 cents are that in Bor we should use forcing author too (as in Clique)
@AskAlexSharov (want to tag Andrew but don't know his tag in gh)

Reply:
Maybe if all test are passed it's right for all consensus?

Reply:
“Engine” is consensus.
« execCfg» it’s not consensus - it’s some our config object - which we created just to combine exec parameters.

@mh0lt - is it bad idea for Bor mining?

Reply:
Yep, this is true. My point was that we use field author in cfg for forcing author in engine. I suppose that we should do it on the mining stage otherwise we can't find the author (if it's not forced it is extracted from Extra header field but in exec stage of mining we are not sealing the block that means that our Extra field would be empty anyway)

> “Engine” is consensus. « execCfg» it’s not consensus - it’s some our config object - which we created just to combine exec parameters.
> 
> @mh0lt - is it bad idea for Bor mining?

Reply:
I'm not sure what author is used for in this context.  Looking at the code is doesn't look like its used at the moment. The logic around checking for valid block producers in bor is done by looking at the header contents:

see: https://github.com/erigontech/erigon/blob/6c387bfbc3dbebd4be681fc0e1cce5a019950603/polygon/bor/bor.go#L508
and: 
https://github.com/erigontech/erigon/blob/6c387bfbc3dbebd4be681fc0e1cce5a019950603/polygon/bor/bor.go#L847

I don't know what the impact of setting it is.

Reply:
As it's not the problem of only Bor @yperbasis what do you think about all others consensus (except bor and clique) should we enforce author in them too?